src/Polly.Core/Simmy/Outcomes/OutcomeChaosStrategy.cs

```diff
@@ -0,0 +1,11 @@
+﻿using Polly.Telemetry;
+
+namespace Polly.Simmy.Outcomes;
+
+internal sealed class OutcomeChaosStrategy : OutcomeChaosStrategy<object>
```

Comment:
Do you need this one?

Reply:
yep, that's used from the [AddFault non-generic APIs](https://github.com/vany0114/Polly/blob/simmy-v8/src/Polly.Core/Simmy/Outcomes/OutcomeCompositeStrategyBuilderExtensions.cs)